readme.md

```diff
@@ -2,6 +2,10 @@
 
 > Awesome tooling and resources in the Chrome DevTools ecosystem
 
+## Learning
+* [Dev Tips](https://umaar.com/dev-tips/) - Large collection of tips as animated gifs.
+* [Discover DevTools](https://www.codeschool.com/courses/discover-devtools) - Introductory course with videos and challenges.
```

Comment:
We aren't really pushing Discover DevTools anymore. It is very dated and doesn't seem like it will be getting any course updates for the new additions and design.

Should probably drop this. @paulirish WDYT?